src/main/main.c

```diff
@@ -38,7 +38,7 @@ int main(void)
     return 0;
 }
 
-void FAST_CODE FAST_CODE_NOINLINE run(void)
+void run(void)
```

Comment:
Is this change needed? This is potentially what is driving ITCM over the brink on F745.

Reply:
Reverted.